queries.sql

```diff
@@ -1,7 +1,9 @@
 -- Part 1: Test it with SQL
-
+SELECT *
```

Comment:
```suggestion
-- id (int, PK), employer(varchar), name(varchar), skills(varchar)
```

The goal of this part for SQL was to add comments that represent the initial columns for the job table. Its structure does change toward the end of the assignment, but in the beginning, there was an employer column. The job table does also contain a name column. So, in this file, you would have written something like `-- employer(varchar), name(varchar)` just to let us know that you saw the initial columns for it.

Not having this doesn't affect your grade or code, though. So, no need to refactor this file. 🙂